docs/source/usage/index.rst

```diff
@@ -284,3 +284,11 @@ Trunk Ports
 .. _Ironic boot-from-volume documentation: https://docs.openstack.org/ironic/latest/admin/boot-from-volume.html
 .. _python-esiclient: https://github.com/CCI-MOC/python-esiclient
 .. _python-esileapclient: https://github.com/CCI-MOC/python-esileapclient
+
+Serial Consoles
+~~~~~~~~~~~~~~~
+
+Ironic nodes can be accessed through configuring a web or console interface with documentation on how to configure 
+found _here: https://docs.openstack.org/ironic/rocky/admin/console.html. Simply choose an available port and pass to a node's driver_info as 'ipmi_terminal_port'.
```

Comment:
I know it's duplicating documentation somewhat, but I think it's better to summarize what users need to do here in this document than to ask them to follow a link outside. The general format for a lot of this is already in this document.

Also, we *definitely* should not be linking to a rocky version of the documentation; at this point that's six versions ago.